mindmeld/components/_config.py

```diff
@@ -464,11 +464,28 @@
     },
 }
 
+DEFAULT_AUGMENTATION_CONFIG = {
```

Comment:
Also, like we discussed, would be nice to expose some of the generation parameters to the developer through the config (especially for the multilingual models)

Reply:
The config now has a params dict that can be populated with configurable model parameters. We should provide a detailed documentation of acceptable parameters in the documentation.

Reply:
Same comment as before. Are we now having just a single set of parameters for both forward and back translations?

Reply:
Added.